### PR TITLE
Fix coverage reports - Closes #276

### LINKED ISSUE #276: Get back coverage reports
Since migration from Travis to Jenkins PRs are (sometimes) lacking the Coverage checks
Also Coverage badge in README.md shows outdated number as coveralls didn't run on development since May 20: https://coveralls.io/github/LiskHQ/lisk-nano?branch=development

### PR DESCRIPTION
**Pull request by slaweet**
Closes #276

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = function (config) {
     preprocessors: {
```
Comment (Isabello) [inline karma.conf.js]: You can remove the jenkins constant, and karma-coveralls(maybe, depends on the way lcov gets made...)

File: Jenkinsfile [hunk 2]
```diff
@@ -76,6 +76,7 @@ node('lisk-nano-01'){
         # Run test
         cd $WORKSPACE
         npm run test
+        cat coverage/*/lcov.info | coveralls -v
```
Comment (Isabello) [inline Jenkinsfile]: add a new line and a comment that we are submitting coverage here.

File: karma.conf.js (hunk 1 above)
Comment (slaweet) [inline karma.conf.js]: `onJenkins` is still used to decide whether generate `html` or `lcov` report.  
I removed the `karma-coveralls`, dependency.

Comment (Isabello) [inline karma.conf.js]: ok great :)

**Pull request merged**